src/commons/sideContent/content/SideContentCseMachine.tsx

```diff
@@ -296,6 +299,15 @@ class SideContentCseMachineBase extends React.Component<CseMachineProps, State>
             </ButtonGroup>
           </div>
         </div>{' '}
+        {this.state.visualization &&
+        this.props.machineOutput.length &&
+        this.props.machineOutput[0].type === 'errors' ? (
+          this.props.machineOutput.map((slice, index) => (
+            <Output output={slice} key={index} usingSubst={false} isHtml={false} />
+          ))
+        ) : (
+          <div></div>
+        )}
```

Comment:
Is the purpose of the empty `div` to show nothing? If yes, does it makes sense to use conditional rendering instead?

```suggestion
        {this.state.visualization &&
        this.props.machineOutput.length &&
        this.props.machineOutput[0].type === 'errors' && (
          this.props.machineOutput.map((slice, index) => (
            <Output output={slice} key={index} usingSubst={false} isHtml={false} />
          ))
        )}
```